acl_setup.php

```diff
@@ -236,7 +258,7 @@
  $gacl->add_acl(
   array(
    'acct'=>array('bill', 'disc', 'eob', 'rep', 'rep_a'),
-   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl'),
+   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','gadd','gcalendar','glog'),
```

Comment:
The mistake you are making here (note this aco/section/group/acl stuff is confusing :) ) is that this entire array is to build the write acl for Administrators. The admin section above is only for aco items that are in the admin section. You need to add another section in this array titled 'groups' and then add your aco elements there.